app/support/reservations/validations.rb

```diff
@@ -6,13 +6,14 @@ module Reservations::Validations
     delegate :editing_time_data, to: :order_detail, allow_nil: true
 
     validates_uniqueness_of :order_detail_id, allow_nil: true
-    validates :product_id, :reserve_start_at, presence: true
```

Comment:
Oh, it worked, I just ended up separating them because I thought it was easier to read.